sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/b2c/AADB2CAuthorizationRequestResolver.java

```diff
@@ -59,13 +60,17 @@ public OAuth2AuthorizationRequest resolve(@NonNull HttpServletRequest request) {
 
     @Override
     public OAuth2AuthorizationRequest resolve(@NonNull HttpServletRequest request, String registrationId) {
-        if (StringUtils.hasText(passwordResetUserFlow) && isForgotPasswordAuthorizationRequest(request)) {
-            final OAuth2AuthorizationRequest authRequest = defaultResolver.resolve(request, passwordResetUserFlow);
-            return getB2CAuthorizationRequest(authRequest, passwordResetUserFlow);
-        }
+        if (StringUtils.hasText(registrationId)) {
```

Comment:
Will putting this if clause outside change the logic of this method?

Reply:
If there's a registrationId named resetPassword, it means there's available registrationId.